package.json

```diff
@@ -27,5 +27,6 @@
         "url": "https://github.com/Microsoft/tslib.git"
     },
     "main": "tslib.js",
+    "jsnext:main": "tslib.es6.js",
```

Comment:
It seems to me that the ideal solution would be to have a single typescript source file which is then compiled to both targets, but I'm not sure how to handle the UMD / global wrapper stuff in the es5 file.